common/STcommon.py

```diff
@@ -67,7 +67,7 @@ def parseArgs():
 	'''Parses args using the argparse lib'''
 	parser = argparse.ArgumentParser(description='Location logging server')
 
-	parser.add_argument('-c', '--configure', nargs=2, metavar='ADDRESS PORT')
+	parser.add_argument('-c', '--configure', nargs=3, metavar='ADDRESS PORT LOGFILE')
```

Comment:
I initially planned on using `-c` to pass the initial configuration from the install script. But if we're holding off on that for now, I could probably get rid of `-c`. client/server.py already accepts `-c` for the address/port, so maybe we only need `-L` in STCommon (or maybe just in client/server.py) for the log file.